$NetBSD$

Just to make compatible with version of Rust, etc I have (1.29.1)

--- ../vendor/fuchsia-cprng-0.1.0/Cargo.toml.orig	1970-01-01 00:00:00.000000000 +0000
+++ ../vendor/fuchsia-cprng-0.1.0/Cargo.toml
@@ -11,7 +11,6 @@
 # will likely look very different (and much more reasonable)
 
 [package]
-edition = "2018"
 name = "fuchsia-cprng"
 version = "0.1.0"
 authors = ["Erick Tryzelaar <etryzelaar@google.com>"]
